olp-cpp-sdk-dataservice-write/src/generated/IngestApi.h

```diff
@@ -109,19 +109,18 @@ class IngestApi {
    * verifies the integrity of your request and prevents modification by a third
    * party.It will be created by the service if not provided. A SHA-256 hash
    * consists of 256 bits or 64 chars.
-   * @param callback IngestSdiiCallback which will be called with the
-   * IngestSdiiResponse when the operation completes.
+   * @param context A CancellationContext, which can be used to cancel request.
    *
    * @return A CancellationToken which can be used to cancel the ongoing
    * request.
```

Comment:
forgot to update the doxygen about the return value

Reply:
Done